coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Leaderboard/LeaderboardEntry.cs

```diff
@@ -26,11 +29,36 @@ public class LeaderboardEntry
         /// <example>John Doe</example>
         public string Name { get; set; }
         
+        /// <summary>
+        /// Leaderboard rank
+        /// </summary>
+        /// <value>Rank</value>
+        /// <example>2</example>
+        public int Rank { get; set; }
+        
         /// <summary>
         /// Account score
         /// </summary>
         /// <value>Account score</value>
         /// <example>25</example>
         public int Score { get; set; }
+
+        protected bool Equals(LeaderboardEntry other)
```

Comment:
Should Equals and GetHashCode have doc comments? I'm getting the following warnings:

Missing XML comment for publicly visible type or member 'LeaderboardEntry.Equals(object)' [CoffeeCard.Models]csharp(CS1591)

Reply:
True that it gives a warning, however I don't think there is much value in documenting a vanilla method, so I have disabled the warning in this class